package/system/procd/files/service

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+main() {
+	local service="$1"
+	local cmd="$2"
+
+	local boot status
+
+	if [ -f "/etc/init.d/${service}" ]; then
+		if [ "$cmd" = "list" ]; then
```

Comment:
This effectively overrides any `list` command from `/etc/init.d/${service}`. I think (I am not an OpenWrt maintainer) it would be better to have this directly in the init script support functions (`/etc/rc.common`), although it would conflict with the `list` command from `lib/functions.sh`, so a different command name would be necessary (like `procd_list`). So that you could call interchangeably `/etc/init.d/my_service procd_list` and `service my_service procd_list`.

Reply:
You're right about that. I'll have to look at that again. Thank you for your feedback. 
I will rebase this